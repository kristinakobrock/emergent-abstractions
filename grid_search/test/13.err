  0%|                                                                                                     | 0/50 [00:00<?, ?it/s]  6%|#####5                                                                                       | 3/50 [00:00<00:02, 23.47it/s] 12%|###########1                                                                                 | 6/50 [00:00<00:02, 20.71it/s] 20%|##################4                                                                         | 10/50 [00:00<00:01, 24.34it/s] 26%|#######################9                                                                    | 13/50 [00:00<00:01, 25.64it/s] 32%|#############################4                                                              | 16/50 [00:00<00:01, 22.90it/s] 38%|##################################9                                                         | 19/50 [00:00<00:01, 22.56it/s] 44%|########################################4                                                   | 22/50 [00:00<00:01, 23.34it/s] 50%|##############################################                                              | 25/50 [00:01<00:01, 24.89it/s] 56%|###################################################5                                        | 28/50 [00:01<00:00, 24.98it/s] 64%|##########################################################8                                 | 32/50 [00:01<00:00, 27.61it/s] 70%|################################################################3                           | 35/50 [00:01<00:00, 26.95it/s] 76%|#####################################################################9                      | 38/50 [00:01<00:00, 27.48it/s] 82%|###########################################################################4                | 41/50 [00:01<00:00, 25.74it/s] 88%|################################################################################9           | 44/50 [00:01<00:00, 25.58it/s] 94%|######################################################################################4     | 47/50 [00:01<00:00, 26.66it/s]100%|############################################################################################| 50/50 [00:02<00:00, 24.32it/s]100%|############################################################################################| 50/50 [00:02<00:00, 24.88it/s]
  0%|                                                                                                     | 0/13 [00:00<?, ?it/s] 15%|##############3                                                                              | 2/13 [00:00<00:00, 19.77it/s] 38%|###################################7                                                         | 5/13 [00:00<00:00, 25.61it/s] 62%|#########################################################2                                   | 8/13 [00:00<00:00, 27.47it/s] 85%|#############################################################################8              | 11/13 [00:00<00:00, 26.10it/s]100%|############################################################################################| 13/13 [00:00<00:00, 27.22it/s]
