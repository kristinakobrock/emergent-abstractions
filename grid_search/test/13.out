# ["--attributes=3", "--values=4", "--game_size=10", "--vocab_size_factor=3", "--n_epochs=60", "--batch_size=64", "--learning_rate=0.001", "--hidden_size=128", "--temp_update=0.99", "--temperature=2"]
Creating train_ds and val_ds...

Creating test_ds...
{"loss": 0.6805167198181152, "acc": 0.6009761691093445, "length": 3.4725632667541504, "mode": "train", "epoch": 1}
{"loss": 0.672679603099823, "acc": 0.6185546517372131, "length": 1.0, "mode": "test", "epoch": 1}
{"loss": 0.6651434898376465, "acc": 0.6211033463478088, "length": 3.298412561416626, "mode": "train", "epoch": 2}
{"loss": 0.6707939505577087, "acc": 0.6173827648162842, "length": 1.0, "mode": "test", "epoch": 2}
{"loss": 0.6622704267501831, "acc": 0.6224381923675537, "length": 2.8783764839172363, "mode": "train", "epoch": 3}
{"loss": 0.667425274848938, "acc": 0.6181640625, "length": 1.0703125, "mode": "test", "epoch": 3}
{"loss": 0.654887318611145, "acc": 0.623763918876648, "length": 3.0781381130218506, "mode": "train", "epoch": 4}
{"loss": 0.6522289514541626, "acc": 0.6178802847862244, "length": 1.84375, "mode": "test", "epoch": 4}
{"loss": 0.6337668299674988, "acc": 0.6428239941596985, "length": 3.1055915355682373, "mode": "train", "epoch": 5}
{"loss": 0.6340916752815247, "acc": 0.6519531607627869, "length": 4.0, "mode": "test", "epoch": 5}
{"loss": 0.6131104230880737, "acc": 0.667567253112793, "length": 3.530825138092041, "mode": "train", "epoch": 6}
{"loss": 0.6334719657897949, "acc": 0.655468761920929, "length": 4.0, "mode": "test", "epoch": 6}
{"loss": 0.594581127166748, "acc": 0.6856293082237244, "length": 3.8097035884857178, "mode": "train", "epoch": 7}
{"loss": 0.6241262555122375, "acc": 0.6650390625, "length": 4.0, "mode": "test", "epoch": 7}
{"loss": 0.5759247541427612, "acc": 0.7013556361198425, "length": 3.8917434215545654, "mode": "train", "epoch": 8}
{"loss": 0.6203633546829224, "acc": 0.6794921159744263, "length": 4.0, "mode": "test", "epoch": 8}
{"loss": 0.5547589659690857, "acc": 0.7180046439170837, "length": 3.9288551807403564, "mode": "train", "epoch": 9}
{"loss": 0.6049855947494507, "acc": 0.6960937976837158, "length": 4.0, "mode": "test", "epoch": 9}
{"loss": 0.5342100858688354, "acc": 0.732826292514801, "length": 3.9517064094543457, "mode": "train", "epoch": 10}
{"loss": 0.5910863280296326, "acc": 0.7048826813697815, "length": 4.0, "mode": "test", "epoch": 10}
{"loss": 0.5211845636367798, "acc": 0.7403085231781006, "length": 3.9570090770721436, "mode": "train", "epoch": 11}
{"loss": 0.5863669514656067, "acc": 0.7126953601837158, "length": 4.0, "mode": "test", "epoch": 11}
{"loss": 0.5049280524253845, "acc": 0.7506457567214966, "length": 3.950700521469116, "mode": "train", "epoch": 12}
{"loss": 0.5857275128364563, "acc": 0.7160155773162842, "length": 4.0, "mode": "test", "epoch": 12}
{"loss": 0.49236130714416504, "acc": 0.7579385638237, "length": 3.9655110836029053, "mode": "train", "epoch": 13}
{"loss": 0.5890771150588989, "acc": 0.7232422232627869, "length": 4.0, "mode": "test", "epoch": 13}
{"loss": 0.47976168990135193, "acc": 0.7686777114868164, "length": 3.9716691970825195, "mode": "train", "epoch": 14}
{"loss": 0.5648325085639954, "acc": 0.7146483659744263, "length": 4.0, "mode": "test", "epoch": 14}
{"loss": 0.47149044275283813, "acc": 0.7745558023452759, "length": 3.973043918609619, "mode": "train", "epoch": 15}
{"loss": 0.5612984895706177, "acc": 0.7269530892372131, "length": 4.0, "mode": "test", "epoch": 15}
{"loss": 0.45646506547927856, "acc": 0.7856619954109192, "length": 3.977384090423584, "mode": "train", "epoch": 16}
{"loss": 0.5487470030784607, "acc": 0.7390625476837158, "length": 4.0, "mode": "test", "epoch": 16}
{"loss": 0.4391707479953766, "acc": 0.7948773503303528, "length": 3.9819791316986084, "mode": "train", "epoch": 17}
{"loss": 0.5390375256538391, "acc": 0.7357421517372131, "length": 4.0, "mode": "test", "epoch": 17}
{"loss": 0.42735788226127625, "acc": 0.8084642291069031, "length": 3.9843266010284424, "mode": "train", "epoch": 18}
{"loss": 0.5181196928024292, "acc": 0.7564453482627869, "length": 4.0, "mode": "test", "epoch": 18}
{"loss": 0.416841596364975, "acc": 0.8178529739379883, "length": 3.9863553047180176, "mode": "train", "epoch": 19}
{"loss": 0.5002065300941467, "acc": 0.7693359851837158, "length": 4.0, "mode": "test", "epoch": 19}
{"loss": 0.4017503261566162, "acc": 0.8239758014678955, "length": 3.9855401515960693, "mode": "train", "epoch": 20}
{"loss": 0.5006348490715027, "acc": 0.7666015625, "length": 4.0, "mode": "test", "epoch": 20}
{"loss": 0.39229971170425415, "acc": 0.8289406299591064, "length": 3.988802194595337, "mode": "train", "epoch": 21}
{"loss": 0.5111727118492126, "acc": 0.763671875, "length": 4.0, "mode": "test", "epoch": 21}
{"loss": 0.38869500160217285, "acc": 0.8296855092048645, "length": 3.988988161087036, "mode": "train", "epoch": 22}
{"loss": 0.5027536749839783, "acc": 0.7701172828674316, "length": 4.0, "mode": "test", "epoch": 22}
{"loss": 0.38137194514274597, "acc": 0.8318159580230713, "length": 3.9921844005584717, "mode": "train", "epoch": 23}
{"loss": 0.472678005695343, "acc": 0.7824219465255737, "length": 4.0, "mode": "test", "epoch": 23}
{"loss": 0.37364840507507324, "acc": 0.8385683298110962, "length": 3.991020441055298, "mode": "train", "epoch": 24}
{"loss": 0.4808894991874695, "acc": 0.7869139909744263, "length": 4.0, "mode": "test", "epoch": 24}
{"loss": 0.36466339230537415, "acc": 0.8407436609268188, "length": 3.9936511516571045, "mode": "train", "epoch": 25}
{"loss": 0.5012603998184204, "acc": 0.7755860090255737, "length": 4.0, "mode": "test", "epoch": 25}
{"loss": 0.36152616143226624, "acc": 0.8389269113540649, "length": 3.994530200958252, "mode": "train", "epoch": 26}
{"loss": 0.47730720043182373, "acc": 0.7908201813697815, "length": 4.0, "mode": "test", "epoch": 26}
{"loss": 0.3584449887275696, "acc": 0.8426748514175415, "length": 3.99556565284729, "mode": "train", "epoch": 27}
{"loss": 0.47537583112716675, "acc": 0.7865234017372131, "length": 4.0, "mode": "test", "epoch": 27}
{"loss": 0.35612863302230835, "acc": 0.8434919714927673, "length": 3.9959793090820312, "mode": "train", "epoch": 28}
{"loss": 0.4798256754875183, "acc": 0.7894531488418579, "length": 4.0, "mode": "test", "epoch": 28}
{"loss": 0.3487268090248108, "acc": 0.8466455340385437, "length": 3.9956119060516357, "mode": "train", "epoch": 29}
{"loss": 0.4697864055633545, "acc": 0.7890624403953552, "length": 4.0, "mode": "test", "epoch": 29}
{"loss": 0.34083953499794006, "acc": 0.8489021062850952, "length": 3.9964990615844727, "mode": "train", "epoch": 30}
{"loss": 0.48881006240844727, "acc": 0.7820312976837158, "length": 4.0, "mode": "test", "epoch": 30}
{"loss": 0.33791810274124146, "acc": 0.855118989944458, "length": 3.99684476852417, "mode": "train", "epoch": 31}
{"loss": 0.49293357133865356, "acc": 0.7798828482627869, "length": 4.0, "mode": "test", "epoch": 31}
{"loss": 0.33834654092788696, "acc": 0.8510475754737854, "length": 3.99737286567688, "mode": "train", "epoch": 32}
{"loss": 0.4826675057411194, "acc": 0.7830076813697815, "length": 4.0, "mode": "test", "epoch": 32}
{"loss": 0.3317011892795563, "acc": 0.8576040267944336, "length": 3.9976229667663574, "mode": "train", "epoch": 33}
{"loss": 0.466482937335968, "acc": 0.7951172590255737, "length": 4.0, "mode": "test", "epoch": 33}
{"loss": 0.32669416069984436, "acc": 0.8577289581298828, "length": 3.997969150543213, "mode": "train", "epoch": 34}
{"loss": 0.4753177762031555, "acc": 0.7929686903953552, "length": 4.0, "mode": "test", "epoch": 34}
{"loss": 0.3187468349933624, "acc": 0.8638149499893188, "length": 3.997739553451538, "mode": "train", "epoch": 35}
{"loss": 0.4756704568862915, "acc": 0.7974609732627869, "length": 4.0, "mode": "test", "epoch": 35}
{"loss": 0.3196251392364502, "acc": 0.8628512024879456, "length": 3.998197078704834, "mode": "train", "epoch": 36}
{"loss": 0.4760103225708008, "acc": 0.7992187142372131, "length": 4.0, "mode": "test", "epoch": 36}
{"loss": 0.3144981563091278, "acc": 0.8648702502250671, "length": 3.9985036849975586, "mode": "train", "epoch": 37}
{"loss": 0.4788074493408203, "acc": 0.7927733659744263, "length": 4.0, "mode": "test", "epoch": 37}
{"loss": 0.31052666902542114, "acc": 0.8659642934799194, "length": 3.998598337173462, "mode": "train", "epoch": 38}
{"loss": 0.47668564319610596, "acc": 0.7974609732627869, "length": 4.0, "mode": "test", "epoch": 38}
{"loss": 0.3023383319377899, "acc": 0.8687169551849365, "length": 3.9982173442840576, "mode": "train", "epoch": 39}
{"loss": 0.48493045568466187, "acc": 0.7964844107627869, "length": 4.0, "mode": "test", "epoch": 39}
{"loss": 0.30223584175109863, "acc": 0.8714847564697266, "length": 3.998640298843384, "mode": "train", "epoch": 40}
{"loss": 0.4670569598674774, "acc": 0.80078125, "length": 4.0, "mode": "test", "epoch": 40}
{"loss": 0.3000483512878418, "acc": 0.8728038668632507, "length": 3.998533010482788, "mode": "train", "epoch": 41}
{"loss": 0.46426650881767273, "acc": 0.8050780892372131, "length": 4.0, "mode": "test", "epoch": 41}
{"loss": 0.294589102268219, "acc": 0.8755639791488647, "length": 3.998344659805298, "mode": "train", "epoch": 42}
{"loss": 0.46834665536880493, "acc": 0.8009766936302185, "length": 4.0, "mode": "test", "epoch": 42}
{"loss": 0.2979518473148346, "acc": 0.8741901516914368, "length": 3.9986460208892822, "mode": "train", "epoch": 43}
{"loss": 0.48400789499282837, "acc": 0.7980470061302185, "length": 4.0, "mode": "test", "epoch": 43}
{"loss": 0.2928588390350342, "acc": 0.8745394349098206, "length": 3.9988999366760254, "mode": "train", "epoch": 44}
{"loss": 0.45616239309310913, "acc": 0.8064451813697815, "length": 4.0, "mode": "test", "epoch": 44}
{"loss": 0.2826441824436188, "acc": 0.8804029822349548, "length": 3.9988625049591064, "mode": "train", "epoch": 45}
{"loss": 0.4620475172996521, "acc": 0.8062500357627869, "length": 4.0, "mode": "test", "epoch": 45}
{"loss": 0.2790488600730896, "acc": 0.8812713623046875, "length": 3.99897837638855, "mode": "train", "epoch": 46}
{"loss": 0.47286951541900635, "acc": 0.806640625, "length": 4.0, "mode": "test", "epoch": 46}
{"loss": 0.27429112792015076, "acc": 0.8857493996620178, "length": 3.998809576034546, "mode": "train", "epoch": 47}
{"loss": 0.45670780539512634, "acc": 0.8166016340255737, "length": 4.0, "mode": "test", "epoch": 47}
{"loss": 0.26980647444725037, "acc": 0.8868963122367859, "length": 3.9987335205078125, "mode": "train", "epoch": 48}
{"loss": 0.45911380648612976, "acc": 0.8146485686302185, "length": 4.0, "mode": "test", "epoch": 48}
{"loss": 0.2649204730987549, "acc": 0.8923006653785706, "length": 3.998838186264038, "mode": "train", "epoch": 49}
{"loss": 0.4449644684791565, "acc": 0.8246093392372131, "length": 4.0, "mode": "test", "epoch": 49}
{"loss": 0.26253700256347656, "acc": 0.8944322466850281, "length": 3.9987874031066895, "mode": "train", "epoch": 50}
{"loss": 0.4670237898826599, "acc": 0.8140625357627869, "length": 4.0, "mode": "test", "epoch": 50}
{"loss": 0.2585141360759735, "acc": 0.8955217003822327, "length": 3.998854398727417, "mode": "train", "epoch": 51}
{"loss": 0.45539283752441406, "acc": 0.8220701813697815, "length": 4.0, "mode": "test", "epoch": 51}
{"loss": 0.25722959637641907, "acc": 0.8961809277534485, "length": 3.99936580657959, "mode": "train", "epoch": 52}
{"loss": 0.4508550763130188, "acc": 0.8193359375, "length": 4.0, "mode": "test", "epoch": 52}
{"loss": 0.2511926293373108, "acc": 0.8990387916564941, "length": 3.998887538909912, "mode": "train", "epoch": 53}
{"loss": 0.46426957845687866, "acc": 0.8132811188697815, "length": 4.0, "mode": "test", "epoch": 53}
{"loss": 0.24754579365253448, "acc": 0.9003138542175293, "length": 3.9993247985839844, "mode": "train", "epoch": 54}
{"loss": 0.4257110357284546, "acc": 0.8333984017372131, "length": 4.0, "mode": "test", "epoch": 54}
{"loss": 0.2434317171573639, "acc": 0.9036571383476257, "length": 3.9994239807128906, "mode": "train", "epoch": 55}
{"loss": 0.44315218925476074, "acc": 0.8343750238418579, "length": 4.0, "mode": "test", "epoch": 55}
{"loss": 0.24213102459907532, "acc": 0.9029587507247925, "length": 3.999335289001465, "mode": "train", "epoch": 56}
{"loss": 0.43568384647369385, "acc": 0.828125, "length": 4.0, "mode": "test", "epoch": 56}
{"loss": 0.2324143350124359, "acc": 0.9047051072120667, "length": 3.999737024307251, "mode": "train", "epoch": 57}
{"loss": 0.4312843680381775, "acc": 0.8371094465255737, "length": 4.0, "mode": "test", "epoch": 57}
{"loss": 0.2326827198266983, "acc": 0.9107331037521362, "length": 3.999546527862549, "mode": "train", "epoch": 58}
{"loss": 0.4227069616317749, "acc": 0.8341797590255737, "length": 4.0, "mode": "test", "epoch": 58}
{"loss": 0.22839365899562836, "acc": 0.9113577604293823, "length": 3.999267339706421, "mode": "train", "epoch": 59}
{"loss": 0.41971415281295776, "acc": 0.8419922590255737, "length": 4.0, "mode": "test", "epoch": 59}
{"loss": 0.22669266164302826, "acc": 0.91241854429245, "length": 3.999835252761841, "mode": "train", "epoch": 60}
{"loss": 0.42409956455230713, "acc": 0.8427734971046448, "length": 4.0, "mode": "test", "epoch": 60}
test accuracy: 0.7466071844100952
